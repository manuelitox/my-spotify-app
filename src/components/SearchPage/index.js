import React, { Fragment } from 'react'

import SearchFormContainer from 'components/generic/SearchForm/container'
import StyledSearchPage from './styles'
import StyledWaveSearchIcon from './stylesWaveIcon'
import { ReactComponent as HeadsetIcon } from 'svgs/headset.svg'

const SearchPage = () => (
  <Fragment>
    <StyledSearchPage>
      <HeadsetIcon />
      <SearchFormContainer />
    </StyledSearchPage>
    <StyledWaveSearchIcon />
  </Fragment>
)

export default SearchPage