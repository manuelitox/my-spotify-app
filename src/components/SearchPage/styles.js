import styled from 'styled-components'

const StyledSearchPage = styled.div`
  background-image: linear-gradient(#3DC9C2, #3D4EC9);
  background-size: cover;
  background-position: center;
  color: white;
  height: 70vh;
  padding: 180px 0;
  width: 100%;
`

export default StyledSearchPage