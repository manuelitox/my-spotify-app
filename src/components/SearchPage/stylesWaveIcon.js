import styled from 'styled-components'

import { ReactComponent as WaveSearchIcon } from 'svgs/bgWave.svg'

const StyledWaveSearchIcon = styled(WaveSearchIcon)`
  height: 60px;
  position: relative;
  top: -2px;
  width: 100%;
  @media only screen and (max-width: ${ props => props.theme.size.small }) {
    height: 40px;
  }
`

export default StyledWaveSearchIcon