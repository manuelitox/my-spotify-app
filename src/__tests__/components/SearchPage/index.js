import React from 'react'
import { shallow } from 'enzyme'

import SearchPage from 'components/SearchPage/'

describe('Component | SearchPage', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <SearchPage />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})