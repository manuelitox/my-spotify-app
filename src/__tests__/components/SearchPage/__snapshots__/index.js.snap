// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | SearchPage should render a component 1`] = `
<Fragment>
  <styled.div>
    <ReactComponent />
    <SearchFormContainer />
  </styled.div>
  <Styled(ReactComponent) />
</Fragment>
`;
